Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -611,9 +611,13 @@ extension GridTabViewController {
         }
 
         let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
-        controller.addAction(UIAlertAction(title: Strings.AppMenuCloseAllTabsTitleString, style: .default, handler: { _ in self.closeTabsForCurrentTray() }),
+        controller.addAction(UIAlertAction(title: .AppMenuCloseAllTabsTitleString,
+                                           style: .default,
+                                           handler: { _ in self.closeTabsForCurrentTray() }),
```

Comment:
nit: styling

Reply:
I'm gonna make the argument that this is much more easily readable than a super long line of code. We should update our style to this format... or really, we should technically be doing it this way already, if we're currently following the Wenderlich style guide, because this is part of that style. 

Sure, we lose searchability, but you never search for `title: .AppMenuCloseAllTablTitleString, style: .default, handler:`. I'm merely updating the style to reflect what should be in usage on parts that I'm actually touching. :)